File: clients/java/java-springboot/src/main/java/xcarpaccio/FeedbackMessage.java
package xcarpaccio;

public class FeedbackMessage {
    public String type;
    public String content;

    @Override
    public String toString() {
        return "FeedbackMessage{" +
                "type='" + type + '\'' +
                ", content='" + content + '\'' +
                '}';
    }

    public FeedbackMessage() {
    }
}

File: clients/php/socket/src/FeedbackMessage.php
<?php

class FeedbackMessage
{
    private $_type;
    private $_content;

    public function __construct(string $type, string $content)
    {
        $this->_type = $type;
        $this->_content = $content;
    }

    public function GetMessage(): string
    {
        return $this->_content;
    }

    public function GetType()
    {
        return $this->_type;
    }
}